src/components/Person/Person.jsx

```diff
@@ -1 +1,41 @@
 // Write code here
+import React from 'react';
+import './person.scss';
+
+export function Person({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) {
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+
+      {(
+        isMarried && sex === 'f' && (
+          <p className="Person__partner">
+            {`My husband's name is ${partnerName}`}
+          </p>
+        )
+      ) || (
+        isMarried && sex === 'm' && (
+          <p className="Person__partner">
+            {`My wife's name is ${partnerName}`}
+          </p>
+        )
+      ) || (
+        <p className="Person__partner">I am not married</p>
+      )}
```

Comment:
Too much going on here, you need to fix it:
1. You don't need to repeat yourself with `<p>` declaration, move your condition inside `p` tag
2. Create a variable for `partnerType` with a value of `wife/husband` and use it in you string
3. Then you can use ternary to decide what text you want to show